firmware/applications/SignBuddy/imu.c

```diff
@@ -0,0 +1,119 @@
+#include "imu.h"
+
+#include "board.h"
+#include "common.h"
+#include "i2c.h"
+#include "logger.h"
+#include "bno055.h"
+
+#define PROCESS_PERIOD_MS    1000
+
+typedef struct {
+  uint32_t                last_ticks;
+  i2c_t                   i2c_instance;
+  struct   bno055_t       bno055;
+  struct   bno055_accel_t bno055_accel_xyz;
+  struct   bno055_mag_t   bno055_mag_xyz;
+  struct   bno055_gyro_t  bno055_gyro_xyz;
+} state_t;
+
+static state_t state;
```

Comment:
why did you rename this to state?

Reply:
i thought it'd be a bit more descriptive.

Reply:
That was my suggestion, I thought it was state in the sensors module too but looks like it's just s

Reply:
it is more descriptive, but it's really just supposed to be a short prefix to encapsulate module variables

Reply:
it was state in sensors once, but s is better I find

Reply:
Okay let's change it back to s then to be consistent